objects.md

```diff
@@ -0,0 +1,118 @@
+# About
+
+This repo needs to represent the following information:
+
+1. Adding/removing versions to channels
+2. Remove or add edges to all channels
+3. Remove or add edges to a specific channel
```

Comment:
not sure why we'd ever remove from a single channel.  Do you have a use-case for that?

Reply:
yes. when we went from rhel 8 to rhel 8.1 we were happy to put the edge in the candidate channel, but not in the stable channel. While we still wanted to have both of the nodes in the stable channel.  Eventually we wanted to put the edge in stable, but we wanted to be able to blacklist it for a while.